tempo_embeddings/text/abstractcorpus.py

```diff
@@ -126,14 +126,39 @@ def set_metadatas(self, key, value):
         for passage in self.passages:
             passage.set_metadata(key, value)
 
-    def to_dataframe(self):
+    def to_dataframe(self, sample_size=None, centroid_based_sample=False):
```

Comment:
[Sonarcloud](https://sonarcloud.io/component_measures?metric=new_coverage&selected=Semantics-of-Sustainability_tempo-embeddings%3Atempo_embeddings%2Ftext%2Fabstractcorpus.py&view=list&pullRequest=91&id=Semantics-of-Sustainability_tempo-embeddings) noted that this function is entirely untested. Would be good to have some tests in `test_corpus.py`.